lib/pros-buttons.coffee

```diff
@@ -0,0 +1,49 @@
+module.exports =
+  addButtons: (toolBar) ->
+    toolBar.addButton {
+      icon: 'folder-add',
+      callback: 'PROS:New-Project',
+      tooltip: 'Create a new PROS Project',
+      iconset: 'fi',
+      priority: 0
+    }
+    toolBar.addButton {
+      icon: 'upload',
+      callback: 'PROS:Upload-Project'
+      tooltip: 'Upload PROS project',
+      iconset: 'fi',
+      priority: 0
+    }
+    toolBar.addButton {
+      icon: 'check',
+      callback: 'PROS:Register-Project',
+      tooltip: 'Register PROS project',
+      iconset: 'fi',
+      priority: 0
+    }
+    toolBar.addButton {
+      icon: 'arrow-circle-up',
+      callback: 'PROS:Upgrade-Project',
+      tooltip: 'Upgrade existing PROS project',
+      iconset: 'fa',
+      priority: 0
+    }
+    toolBar.addButton {
+      icon: 'eye-slash',
+      callback: 'PROS:Toggle-Terminal',
+      tooltip: 'Toggle PROS terminal output visibility',
+      iconset: 'fa',
+      priority: 0
+    }
+    # separate power button from the rest of the toolbar to reduce likelihood of it being hit by accident

```

Comment:
We'll go ahead and add this to the status bar, and not part of the tool bar, so can remove the power button from the tool bar